src/rviz/visualizer_app.cpp

```diff
@@ -227,6 +227,8 @@ bool VisualizerApp::init(int argc, char** argv)
 
     load_config_service_ =
         private_nh.advertiseService("load_config", &VisualizerApp::loadConfigCallback, this);
+    load_config_discarding_changes_service_ =
+        private_nh.advertiseService("load_config_discarding_changes", &VisualizerApp::loadConfigDiscardingChangesCallback, this);
```

Comment:
Variables and functions get rather long now. I suggest:
- `load_config_discarding_changes_service_` -> `load_config_discarding_service_`
- `loadConfigDiscardingChangesCallback` -> `loadConfigDiscardingCallback`

Reply:
I agree, and have implemented your suggestions.